Realm/RLMSyncUtil.mm

```diff
@@ -45,8 +45,12 @@
 
 using namespace realm;
 
-static_assert((int)RLMClientResetModeManual == (int)realm::ClientResyncMode::Manual);
 static_assert((int)RLMClientResetModeDiscardLocal == (int)realm::ClientResyncMode::DiscardLocal);
+static_assert((int)RLMClientResetModeDiscardUnsyncedChanges == (int)realm::ClientResyncMode::DiscardLocal);
```

Comment:
Things using deprecated symbols need to be wrapped in
```
#pragma clang diagnostic push
#pragma clang diagnostic ignored "-Wdeprecated-declarations"
// thing
#pragma clang diagnostic pop
```

Reply:
Thanks. From what I searched there is no equivalent for swift :/